<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1734</width>
    <height>1152</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTreeWidget" name="fileTree">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>211</width>
     <height>1152</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>文件名</string>
    </property>
   </column>
   <item>
    <property name="text">
     <string>/</string>
    </property>
   </item>
  </widget>
  <widget class="QTextEdit" name="fileContent">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>151</y>
     <width>991</width>
     <height>901</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="fileName">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>104</y>
     <width>971</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>没有打开文件</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="path">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>251</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="newFile">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>30</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>输入：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="open">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>30</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteFile">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>30</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>1090</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>1090</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1300</x>
     <y>320</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>当前访问磁道：</string>
   </property>
  </widget>
  <widget class="QLabel" name="trackNumber">
   <property name="geometry">
    <rect>
     <x>1520</x>
     <y>320</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>暂未访问</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1300</x>
     <y>410</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>当前访问磁盘块：</string>
   </property>
  </widget>
  <widget class="QLabel" name="blockNumber">
   <property name="geometry">
    <rect>
     <x>1520</x>
     <y>410</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>暂未访问</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>1300</x>
     <y>210</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>磁盘调度算法：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="diskOption">
   <property name="geometry">
    <rect>
     <x>1520</x>
     <y>220</y>
     <width>87</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>FCFS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SCAN</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CSCAN</string>
    </property>
   </item>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
